pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>com.okta</groupId>
         <artifactId>okta-parent</artifactId>
-        <version>1</version>
+        <version>2</version>
```

Comment:
Curious: Do you need to specify a `SNAPSHOT` tag?

<https://github.com/okta/okta-java-parent/blob/master/pom.xml#L28>

Or is this convention?

Reply:
v2 is already released